src/status_im2/contexts/share/view.cljs

```diff
@@ -0,0 +1,168 @@
+   #(rf/dispatch [:share/show-successfully-copied-toast post-copy-message])
+   250))
+
+(defn header
+  []
+  [rn/view
+   [quo/button
+    {:icon                true
+     :type                :blur-bg
+     :size                32
+     :accessibility-label :close-activity-center
+     :override-theme      :dark
+     :style               style/header-button
+     :on-press            #(rf/dispatch [:hide-popover])}
+    :i/close]
+   [quo/text
+    {:size   :heading-1
+     :weight :semi-bold
+     :style  style/header-heading}
+    (i18n/label :t/share)]])
+
+(defn profile-tab
+  [window-width]
+  (let [multiaccount              (rf/sub [:multiaccount])
+        emoji-hash                (string/join "" (get multiaccount :emoji-hash))
+        qr-size                   (- window-width 64)
+        public-pk                 (get multiaccount :public-key)
+        profile-qr-url            (str "https://join.status.im/u/" public-pk)
+        port                      (rf/sub [:mediaserver/port])
+        key-uid                   (get multiaccount :key-uid)
+        emoji-hash-max-width      (* window-width 0.76)
+        link-to-profile-max-width (* window-width 0.70)
+       ]
+    [:<>
+     [rn/view {:style style/qr-code-container}
+      [qr/user-profile-qr-code
+       {:window-width window-width
+        :key-uid      key-uid
+        :qr-size      qr-size
+        :public-key   public-pk
+        :port         port}]
+      [rn/view {:style (style/profile-address-container qr-size)}
+       [rn/view {:style style/profile-address-column}
+        [quo/text
+         {:size   :paragraph-2
+          :weight :medium
+          :style  style/profile-address-label}
+         (i18n/label :t/link-to-profile)]
+        [rn/touchable-highlight
+         {:active-opacity   1
+          :underlay-color   colors/neutral-80-opa-1-blur
+          :background-color :transparent
+          :on-press         #(copy-text-and-show-toast
+                              {:text-to-copy      profile-qr-url
+                               :post-copy-message (i18n/label :t/link-to-profile-copied)})
+          :on-long-press    #(copy-text-and-show-toast
+                              {:text-to-copy      profile-qr-url
+                               :post-copy-message (i18n/label :t/link-to-profile-copied)})}
+         [quo/text
+          {:style           (style/profile-address-content link-to-profile-max-width)
+           :size            :paragraph-1
+           :weight          :medium
+           :ellipsize-mode  :middle
+           :number-of-lines 1}
+          profile-qr-url]]]
+       [rn/view {:style style/share-button-container}
+        [quo/button
+         {:icon                true
+          :type                :blur-bg
+          :size                32
+          :accessibility-label :link-to-profile
+          :override-theme      :dark
+          :on-press            (fn []
+                                 (js/setTimeout
```

Comment:
why timeout?

Reply:
got rid of it : https://github.com/status-im/status-mobile/pull/15528/commits/7fa449a0e6d5ee0ed5d2e229f043c04d8f16e8dd